streams/rs.m3u

```diff
@@ -81,3 +81,29 @@ http://109.92.29.10:1935/tvduga/tvduga/playlist.m3u8
 https://vod1.laki.eu/live/hram/index.m3u8
 #EXTINF:-1 tvg-id="TVPirot.rs" status="online",TV Pirot (240p) [Not 24/7]
 https://5bc45691ca49f.streamlock.net/tvpirot/uzivo/playlist.m3u8
+#EXTINF:-1 tvg-id="EuronewsSerbia.rs",Euronews Serbia (1080p)
```

Comment:
Please move to https://github.com/iptv-org/iptv/blob/master/streams/fr_euronews.m3u

Thanks

Reply:
Done